models/evc.py

```diff
@@ -825,7 +830,17 @@ def should_deploy(self, path=None):
 
         return False
 
-    def deploy_to_path(self, path=None):  # pylint: disable=too-many-branches
+    def error_hapenned(self):
+        """Error ocurred and this EVC is malformed."""
+        self.deactivate()
+        self.disable()
+        self.error_status = True
+
+    def error_deleted(self):
```

Comment:
I'll recommend that you try to name it more descriptively for what it does, maybe `set_no_errors` or `clean_errors` or something similar?

Reply:
Changed with `clean_errors`